internal/pkg/describe/uri.go

```diff
@@ -198,6 +199,16 @@ func (d *BackendServiceDescriber) URI(envName string) (URI, error) {
 			AccessType: URIAccessTypeNone,
 		}, nil
 	}
+	scDNSNames, err := svcDescr.ServiceConnectDNSNames()
+	if err != nil {
+		return URI{}, fmt.Errorf("retrieve service connect DNS names: %w", err)
+	}
+	if len(scDNSNames) != 0 {
+		return URI{
```

Comment:
is it possible that the service has both sc and sd? should we fetch uris for both?

Reply:
Yes it is possible. I *think* it's probably fine to show SC instead of SD in this case? Because if they have SC enabled they can just use SC. Like internal ALB when it is enabled we don't show SD as well.